lib/api_browser/app/views/type/details.html

```diff
@@ -1,10 +1,13 @@
 ﻿<div class="row" ng-if="type">
   <div class="col-lg-12">
-    <h2>Schema</h2>
-    <p ng-if="type.identifier"><b>Media-type identifier: {{ type.identifier }}</b></p>
-    <div ng-if="type.description" ng-bind-html="type.description | markdown"></div>
+    <div ng-if="type.attributes">
+      <h2>Schema</h2>
+      <p ng-if="type.identifier"><b>Media-type identifier: {{ type.identifier }}</b></p>
+      <div ng-if="type.description" ng-bind-html="type.description | markdown"></div>
 
-    <attribute-table attributes="type.attributes"></attribute-table>
+      <attribute-table attributes="type.attributes"></attribute-table>
+    </div>
+    <div ng-if="type.attributes === undefined" type-placeholder type="type" template="standalone" name="type.name" details="type"></div>
```

Comment:
no, it's used by the controller page to show the default media_type. That can probably be rearranged but for now I'd rather not move more things around.